viewing_party/party.py

```diff
@@ -1,23 +1,147 @@
 # ------------- WAVE 1 --------------------
-
+from ast import AnnAssign
+from curses import use_default_colors
+from operator import ne
+import re
+import copy
+# from tkinter import N
 def create_movie(title, genre, rating):
-    pass
+    dic = {}
+    dic["title"] = title
+    dic["genre"] = genre
+    dic["rating"] = rating
+    if dic["title"] == None or dic["genre"] == None or dic["rating"] == None:
+        return None
+    return dic
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(janes_data, MOVIE_TITLE_1):

```

Comment:
The logic here is correct, though the specification was not followed completely. It states...

> Create a function named `watch_movie`. This function should...
> take two parameters: `user_data, title`

In the function definition, a `title` parameter should take the place of `MOVIE_TITLE_1`. We will need to perform checks if the `title` parameter exists in the user's watchlist, then remove and append to the corresponding lists.